components/markets/MarketScroll.tsx

```diff
@@ -59,58 +69,36 @@ const MarketScroll = observer(
       hasReachedEnd || cardWidth * markets.length < containerWidth;
 
     return (
-      <div ref={containerRef} className="flex flex-col">
-        <div className="flex items-center mb-ztg-30">
-          <div className=" font-bold text-[28px]">{title}</div>
-          <div className="flex ml-auto items-center">
-            {showMarketsLink && (
-              <Link
-                href="markets"
-                className="text-ztg-14-150 border-2 border-pastel-blue rounded-[5px] px-[10px] py-[3px]"
-              >
-                Go To Markets
-              </Link>
-            )}
-            <button
-              onClick={handleLeftClick}
-              className={`hidden sm:flex items-center justify-center w-[26px] h-[26px] rounded-full ml-[12px] mr-[8px] ztg-transition ${
-                leftDisabled
-                  ? "bg-geyser text-pastel-blue"
-                  : "bg-pastel-blue text-white"
-              }`}
-              disabled={leftDisabled}
-            >
-              <ChevronLeft className="relative right-[1px]" />
-            </button>
-            <button
-              onClick={handleRightClick}
-              className={`hidden sm:flex items-center justify-center w-[26px] h-[26px] rounded-full ztg-transition  ${
-                rightDisabled
-                  ? "bg-geyser text-pastel-blue"
-                  : "bg-pastel-blue text-white"
-              }`}
-              disabled={rightDisabled}
-            >
-              <ChevronRight className="relative left-[1px]" />
-            </button>
-          </div>
-        </div>
-        <div className="relative">
+      <div ref={containerRef} className="grid sm:grid-cols-2 gap-4 md:gap-6">
+        <h3 className="sm:col-span-1 font-bold text-[28px]">{title}</h3>
+        <HorizontalScroll
+          classes="order-2 sm:order-none"
+          link={link}
+          cta="Go To Markets"
```

Comment:
Think `cta` should also be passed as a prop to the component. Maybe use a specific type for both `cta` and `link`.